corehq/apps/locations/static/locations/js/location.js

```diff
@@ -1,11 +1,17 @@
-/* globals TEMPLATE_STRINGS */
+/* globals django */
 // for product and user per location selection
-hqDefine("locations/js/locations", function() {
+hqDefine("locations/js/location", function() {
+    var initialPageData = hqImport('hqwebapp/js/initial_page_data');
+    var LocationModels = hqImport('locations/js/location_drilldown');
     var insert_new_user = function(user) {
         var $select = $('#id_users-selected_ids');
         $select.multiSelect('addOption', { value: user.user_id, text: user.text });
         $select.multiSelect('select', user.user_id);
     };
+    var TEMPLATE_STRINGS = {
+        new_user_success: _.template(gettext("User " + initialPageData.get('user_name') +
```

Comment:
@millerdev Will this translate correctly or does the `gettext` call need to be one solid string (this used to be something like `gettext("User <%= name %> added successfully...")`)? Can't remember how `gettext` works.

Reply:
> Will this translate correctly...?

No, it will not because the string passed to `gettext()` will be something like `"User dmiller@xyz.com added successfully..."`, which will not be in the translation dictionary. We should continue to use the same template string as before if possible.